modules/swift/examples/code_snippets/SampleCodeTest.swift

```diff
@@ -21,16 +21,28 @@ import CouchbaseLiteSwift
 import MultipeerConnectivity
 import CoreML
 
-
 class SampleCodeTest {
-
+    /**
+     For consistency in code snippets:
+     1. we will use `self.database`/ `database` for database, query, replicator-db related code snippets.
+     2. we will use `self.otherDB` / `otherDB` for listener-db
```

Comment:
this was how I thought, since users can understand the difference. For example, when a replicator connects from a `database` to `otherDB`.

Reply:
OK